app/src/main/res/layout/empty_view_layout.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center">
+
+    <TextView
+        android:id="@+id/txtvtitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingLeft="10dp"
+        android:paddingRight="10dp"
+        android:text="empty"

```

Comment:
Please replace with `tools:text` because otherwise Lint complains about hardcoded strings. (You need to add `xmlns:tools="http://schemas.android.com/tools"` to the LinearLayout to register the tools namespace)